Source/Core/DolphinWX/Main.cpp

```diff
@@ -108,8 +107,16 @@ bool DolphinApp::OnInit()
   wxHandleFatalExceptions(true);
 #endif
 
+#ifdef _WIN32
+  AttachConsole(ATTACH_PARENT_PROCESS);
```

Comment:
@BhaaLseN I agree, but I'm also suspecting the usage of ```freopen``` to prevent cygwin to output anything as @ligfx said.